package.json

```
@@ -26,24 +26,23 @@
   "dependencies": {
     "chalk": "^2.4.1",
     "cosmiconfig": "^5.0.0",
-    "create-jest-runner": "^0.4.1",
+    "create-jest-runner": "^0.5.3",
     "eslint": "^5.6.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.1.0",
     "@babel/core": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^23.6.0",
+    "babel-jest": "^24.8.0",
     "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-config-prettier": "^3.1.0",
+    "eslint-config-prettier": "^4.3.0",
     "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^21.22.1",
-    "eslint-plugin-prettier": "^2.6.2",
+    "eslint-plugin-jest": "^22.6.4",
+    "eslint-plugin-prettier": "^3.1.0",
     "execa": "^1.0.0",
-    "jest": "^23.6.0",
+    "jest": "^24.8.0",
```

This should match the peer dep range verbatim 
```suggestion
    "jest": "^21.0.0 || ^22.0.0 || ^23.0.0 || ^24.0.0",
```
so that `npm ls` passes when testing older jest versions